hoot-core/src/main/cpp/hoot/core/io/HootApiDbBulkInserter.cpp

```diff
@@ -59,7 +59,7 @@ HootApiDbBulkInserter::HootApiDbBulkInserter() : OsmApiDbBulkInserter()
   setConfiguration(conf());
 
   _changesetTags["bot"] = "yes";
-  _changesetTags["created_by"] = "hootenanny";
+  _changesetTags["created_by"] = HOOT_PACKAGE_NAME;
```

Comment:
I found that OSM and OSC files had different "spellings" of Hootenanny in the output files, some with capital H and some lower case.  Anyway I updated a lot of places to use a constant "hootenanny" throughout the output code.